src/hostedPages/layouts/HostedPageLayout/LoadActivityPlaceholder/loadActivityPlaceholder.module.scss

```
@@ -0,0 +1,45 @@
+@use '../../../../globalStyles';
+
+  .loading_indicators {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    width: 100%;
+  }
+  
+  .loading_indicator {
+    width: 100%;
+    height: 3.125rem;
+    border-radius: 0.625rem; 
+    margin: 0.4rem 0;
+    background: linear-gradient(90deg, transparent, #EEEEEE, transparent);
+    background-size: 200% 100%;
+    background-position: -100% 0;
+    animation: shimmer 1.2s infinite;
+  }
+
+  /* Styles for larger screens */
+  @media (min-width: 768px) {
+    .loading_indicators {
+       max-width: 570px;
+       padding: 2rem 0;
+    }
+  }
+   
+   /* Styles for smaller screens (mobile) */
+   @media (max-width: 767px) {
+    .loading_indicators {
+       min-width: 336px;
+       padding: 2rem 1rem;
+    }
+   }
```

I think you could just do this:


```suggestion
.loading_indicators {
  max-width: var(--awell-activity-max-width);
  padding: var(--awell-spacing-8) var(--awell-spacing-4);
}
```

---

And JFYI

This
```scss
  @media (min-width: 768px) {
    .loading_indicators {
       max-width: 570px;
       padding: 2rem 0;
    }
  }
   
   /* Styles for smaller screens (mobile) */
   @media (max-width: 767px) {
    .loading_indicators {
       min-width: 336px;
       padding: 2rem 1rem;
    }
   }
```

Can also be written as just this:

```scss
.loading_indicators {
  min-width: 336px;
  padding: 2rem 1rem;

  @media (min-width: 768px) {
     max-width: 570px;
     padding: 2rem 0;
  }
}
```

---

this also doesn't quite work. If I had more time I would try to condense it but I rather merge it now and move to testing.